test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/AichatwebTemplatesTests.cs

```diff
@@ -34,16 +33,28 @@ public async Task BasicTest()
         // Get the template location
         string templateLocation = Path.Combine(TemplateFeedLocation, "Microsoft.Extensions.AI.Templates", "src", "ChatWithCustomData");
 
-        // Treat *.in files as text, see https://github.com/VerifyTests/EmptyFiles#istext
-        FileExtensions.AddTextExtension(".in");
```

Comment:
Excellent; great that this works. I think you're right that this will yield more intuitive snapshots with less risk of us leaking something into the actual template package that wasn't expected.